test/games/am2r/generator/test_am2r_pool_creator.py

```diff
@@ -52,14 +52,9 @@ def test_am2r_pool_creator(am2r_game_description, preset_manager):
         c("The Nest", "Hideout Omega Nest", "Pickup (Omega)"),
         c("Industrial Complex", "Lower Factory Gamma Nest", "Pickup (Gamma)"),
         c("The Nest", "Depths Omega Nest East", "Pickup (Omega)"),
-        c("The Tower", "Exterior Zeta Nest West", "Pickup (Zeta)"),
-        c("Main Caves", "Mines Alpha Nest", "Pickup (Alpha)"),
-        c("The Tower", "Gamma Nest South East", "Pickup (Gamma)"),
-        c("The Nest", "Depths Omega Nest South West", "Pickup (Omega)"),
-        c("Hydro Station", "Exterior Alpha Nest", "Pickup (Alpha)"),
     ]
 
-    assert len(results.to_place) == wl.num_pickup_nodes - 31 - len(results.assignment)
+    assert len(results.to_place) == wl.num_pickup_nodes - 36 - len(results.assignment)
```

Comment:
you can safely use `len(locations)` here instead of a magic number since you're already asserting that the contents of the list are what you expect